cmd/nerdctl/container/container_commit.go

```diff
@@ -98,3 +107,10 @@ func commitShellComplete(cmd *cobra.Command, args []string, toComplete string) (
 	}
 	return nil, cobra.ShellCompDirectiveNoFileComp
 }
+
+func verifyOption(op types.ContainerCommitOptions) (bool, error) {
```

Comment:
This does not deserve to be a separate function

Reply:
drop function